packages/core/src/checkout/checkout-service.ts

```diff
@@ -1399,6 +1406,33 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'extensions' });
     }
 
+    /**
+     * Manages the command handler for an extension.
+     *
+     * @alpha
+     * @param extensionId - The ID of the extension sending the command.
+     * @param command - The command to be handled.
+     * @param handler - The handler function for the extension command.
+     * @returns A function that, when called, will deregister the command handler.
+     */
+    listenExtensionCommand(
+        extensionId: string,
+        command: ExtensionCommand,
+        handler: ExtensionCommandHandler,
+    ): () => void {
+        return this._extensionMessenger.listen(extensionId, command, handler);
+    }
+
+    /**
+     * Stops handling commands for an extension.
+     *
+     * @alpha
+     * @param extensionId - The ID of the extension that originally sent the commands.
+     */
+    stopListenExtensionCommand(extensionId: string): void {
```

Comment:
Maybe we could skip this method for now unless we really need it. `IframeEventListener#stopListen` doesn't actually remove the references to the handlers previously registered. So we need to do a bit of work to actually do what we want.

Reply:
Removed. We can circle back to it once we have the use case.